apps/etterlatte-saksbehandling-ui/server/src/config/config.ts

```diff
@@ -23,6 +23,12 @@ function hentApiConfigFraEnv() {
   }
 }
 
+export const FeatureToggleConfig = {
+  uri: process.env.UNLEASH_URL || 'https://unleash.nais.io/api/',
+  cluster: process.env.NAIS_CLUSTER_NAME || 'dev-gcp',
+  applicationName: process.env.NAIS_APP_NAME || 'etterlatte-saksbehandling-ui',
```

Comment:
Hvordan vil dette fungere med lokal kjøring (`etterlatte-saksbehandling-ui-lokal`)?

Reply:
Vil fortsatt kalle unleash men med fallback verdier.

Har aldri brukt lokal - så er ikke helt sikker på forventninger her

Reply:
Ville testet hele greia med lokal kjøring. Hvis det ikke fungerer ødelegger man for ganske mange på teamet som bruker lokal kjøring regelmessig.

Reply:
Snakket med @Watercolours - min lokal testing var faktisk gjort med det han mente med lokal kjøring :)